### DISCUSSION TITLE
janusgraph how to connect the spark on k8s

### DISCUSSION THREAD

**Discussion by laichen2021** [JanusGraph/janusgraph#3066] [Q&A]
My spark and janusgraph are deployed by k8s, but when I connect to the spark from janusgraph, it failed. 
![image](https://user-images.githubusercontent.com/94055492/171561900-3c0dc49a-1632-435f-abe4-8ad6ac5e0306.png)

**Comment by vtslab** [comment]
This is not supported yet: the lib folder of the janusgraph distribution does not contain the spark-kubernetes jar. Neither a custom build of janusgraph with the dependency added or adding jars manually by trial and error seem attractive. I also doubt whether it is worth the effort: OLAP with JanusGraph is mostly disapointing.
